docs/notes/2.26.x.md

```diff
@@ -12,51 +12,45 @@ Thank you to [Klaviyo](https://www.klaviyo.com/) and [Normal Computing](https://
 
 ### Highlights
 
-New kubernetes backend! See [docs](https://www.pantsbuild.org/stable/docs/kubernetes) for details.
-
-New `python_format_string` codegen for the kubernetes backend! See [docs](https://www.pantsbuild.org/stable/docs/kubernetes) for details.
-
-### Remote caching/execution
-
-- Remote cache: `FindMissingBlobsRequest` will now make multiple request if the number of files is large. (https://github.com/pantsbuild/pants/pull/20708)
-
-### Deprecations
-
+- New kubernetes backend
+- Upgrades of the default versions for many tools
+- Publishing docker images non-interactively and in parallel
+- Much faster Ruff invocations on macOS
 
 ### General
 
-- [Fixed](https://github.com/pantsbuild/pants/pull/21959) a bug where exponential backoff of file downloads could wait up to 18 days between retries under the default settings. [The `[GLOBAL].file_downloads_retry_delay ` option](https://www.pantsbuild.org/2.26/reference/global-options#file_downloads_retry_delay) is now used as a multiplier (previously it was the exponential base), and so any customisation of this option may need reconsideration for the new behaviour.
-- [Fixed](https://github.com/pantsbuild/pants/pull/22024) handling of non-absolute paths in PATH (see [#21954](https://github.com/pantsbuild/pants/issues/21954) for more info).
-- [Fixed](https://github.com/pantsbuild/pants/pull/22059) `pex_binary.complete_platforms` can now accept a codegened source, such as the result of an `shell_command`.
+[Fixed](https://github.com/pantsbuild/pants/pull/21959) a bug where exponential backoff of file downloads could wait up to 18 days between retries under the default settings. [The `[GLOBAL].file_downloads_retry_delay ` option](https://www.pantsbuild.org/2.26/reference/global-options#file_downloads_retry_delay) is now used as a multiplier (previously it was the exponential base), and so any customisation of this option may need reconsideration for the new behaviour.
```

Comment:
nit: Isn't exponential backoff supposed to be.. exponential?  Should this be something like  "previously the delay growth was super-exponential"?  (Not sure of the precise math term.)

Reply:
It was and is still exponential, i.e. if $n$ is the number of attempts, both old and new had maximum wait times of $A \cdot B^n$ for some $A$, $B$... but $A$ and $B$ were wrong: 

- previously we used `file_downloads_retry_delay` as the base $B$ (and $A = 1$). This grows muuuccchhh faster than intended, e.g. for `file_downloads_retry_delay = 200` (nominally ms), the old scheme gives maximum waits of 200ms, 40000ms, 8000000ms, 1600000000ms.
- now we use `file_downloads_retry_delay` as the multiplier $A$ and $B = 2$, giving maximum waits of 200ms, 400ms, 800ms, 1600ms